index.js

```diff
@@ -2,6 +2,10 @@ const loopWhile = require('deasync').loopWhile
 const processor = require('./processor')
 
 module.exports = (css, settings) => {
+  const cssWithPlaceholders = css
+    .replace(/%%styled-jsx-(expression|placeholder)-(\d+)%%/g, (_, p, id) =>
```

Comment:
I made a mistake and never meant to use `expression` here. It should be `%%styled-jsx-placeholder-(\d+)%%
Can you revert to the original version and just replace `expression-` with `placeholder-`?